libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginViewModel.kt

```diff
@@ -123,6 +124,9 @@ open class LoginViewModel(val bootConfig: BootConfig): ViewModel() {
 
 
     init {
+        // Get isIDPLoginFlowEnabled from SDK manager
+        isIDPLoginFlowEnabled.value = SalesforceSDKManager.getInstance().isIDPLoginFlowEnabled
```

Comment:
NIT:  Can this just be added to the declaration?   And does it need to be "mutable"?  Would something like:
```
internal val isIDPLoginFlowEnabled = derivedStateOf { SalesforceSDKManager.getInstance().isIDPLoginFlowEnabled }
```
allow the value to update if/when the data source does?

Reply:
Good idea. Done here: https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2658/commits/621480890d4b2c28187f4194789ba3a49689556c